lib/internal/crypto/ec.js

```diff
@@ -122,12 +122,11 @@ function createECRawKey(namedCurve, keyData, isPublic) {
       break;
   }
 
-  if (isPublic) {
-    handle.initEDRaw(namedCurve, keyData, kKeyTypePublic);
-    return new PublicKeyObject(handle);
+  const keyType = isPublic ? kKeyTypePublic : kKeyTypePrivate;
+  if (!handle.initEDRaw(namedCurve, keyData, keyType)) {
+    throw lazyDOMException('Failure to generate key object');
   }
 
-  handle.initEDRaw(namedCurve, keyData, kKeyTypePrivate);
```

Comment:
lacking the `new PublicKeyObject` return in this updated block

Reply:
oops, thanks!